posthog/api/capture.py

```diff
@@ -268,7 +269,12 @@ def drop_events_over_quota(token: str, events: List[Any]) -> List[Any]:
             if token in limited_tokens_recordings:
                 EVENTS_DROPPED_OVER_QUOTA_COUNTER.labels(resource_type="recordings", token=token).inc()
                 if settings.QUOTA_LIMITING_ENABLED:
-                    continue
+                    # tell clients to wait until midnight before trying again, or at least 1 hour, if midnight is soon
+                    seconds_until_midnight = max(
+                        (datetime.now() + timedelta(days=1)).replace(hour=0, minute=0, second=0) - datetime.now(),
+                        timedelta(hours=1),
+                    ).total_seconds()
+                    raise Throttled(detail="Recording event dropped due to billing limit", wait=seconds_until_midnight)
```

Comment:
If we raise here this means we wouldn't process any events... You might be throttled on one quota but not another?

Also why do we only raise for recordings and not for events?

Not sure what the best approach is here but I'm wondering if we should potentially move to a payload response instead. i.e. 
1. Detect if events are being quota limited or recordings
2. Respond with a 200 but with content saying something like
```
{
 errors: [
  'QUOTA_LIMITED_EVENTS',
  'QUOTA_LIMITED_RECORDINGS',
 ]
}
```

Then delay on the frontend for some fixed time (10 minutes), which will anyway get ignored after a reload.

Reply:
So, my understanding was that quota limiting is EE only. And, so _really_ cloud only. And we've split recordings and events.

So, really we only _need_ to support all recordings or all events... no?

Can support mixed payloads but it pushes more complexity around...